vscode/package.json

```diff
@@ -113,6 +113,11 @@
           "type": "boolean",
           "default": "true",
           "description": "Enables the Q# formatter."
+        },
+        "Q#.showCircuitCodeLens": {
+          "type": "boolean",
+          "default": false,
```

Comment:
Do you need to provide a default in the package.json and then a default value when reading the property in code also? I noticed this when Scott added the formatting setting also.

Not important. Just seems odd to have a 'default' in both places.

Reply:
(And as per the comment in the code that reads it, we should default to 'true' while this is going out only in dev builds)

Reply:
Yeah, I was copying his example. The annoying thing is that `vscode.workspace.getConfiguration("Q#").get<boolean>("showCircuitCodeLens")` technically returns `boolean | undefined`, so I went with the lesser of two evils between adding an `!` and specifying a default value. At least I'll add a comment to clarify.